Makefile

```diff
@@ -33,6 +37,7 @@ $(PROTODIR):
 clean:
 	$(MAKE) -C $(PROTODIR) clean
 	$(RMRF) node
+	$(RMRF) gitaly
```

Comment:
@stepanvalyavskiy the output binary will be `degitx-gitaly` after changing directory name.